backend/src/election/mod.rs

```diff
@@ -90,3 +91,43 @@ pub async fn election_status(
     let statuses = polling_station_repo.status(id).await?;
     Ok(Json(ElectionStatusResponse { statuses }))
 }
+
+/// Download a generated PDF
+#[utoipa::path(
+    get,
+    path = "/api/elections/{election_id}/download-pdf",
+    responses(
+        (
+            status = 200,
+            description = "PDF",
+            content_type="text/pdf; charset=utf-8",
+            headers(
+                ("Content-Type", description = "text/pdf; charset=utf-8"),
+                ("Content-Disposition", description = "attachment; filename=\"filename.pdf\"")
+            )
+        ),
+        (status = 404, description = "Not found", body = ErrorResponse),
+        (status = 500, description = "Internal server error", body = ErrorResponse),
+    ),
+    params(
+        ("election_id" = u32, description = "Election database id"),
+    ),
+)]
+pub async fn election_download_pdf(
+    State(elections_repo): State<Elections>,
+    Path(id): Path<u32>,
+) -> Result<(HeaderMap, Vec<u8>), APIError> {
+    let _election = elections_repo.get(id).await?;
+
+    // TODO: Replace this with the generated PDF
+    let filename = "proces-verbaal.pdf";
+    let content = &[];
+
+    let disposition_header = format!("attachment; filename=\"{}\"", filename);
+
+    let mut headers = HeaderMap::new();
+    headers.insert(header::CONTENT_TYPE, "text/pdf; charset=utf-8".parse()?);
```

Comment:
The MIME type for PDF is `application/pdf` without any charset. (See [MDN Common MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)).

Reply:
[58f0f1e](https://github.com/kiesraad/abacus/commit/58f0f1eee51de154409f9e2784ca8d2e54632c57) addresses this